.github/ISSUE_TEMPLATE/DOCUMENTATION.md

```diff
@@ -0,0 +1,53 @@
+---
+name: "📜📖📄📘 📚  Documentation enhancement"
+about: Suggest edits for Wagtail documentation
+title: ''
+labels: 'type:Documentation'
+assignees: ''
+
+---
+
+<!--
+  Summarize the documentation change you're suggesting in the Issue title.
+-->
+
+### Pertinent section of the Wagtail docs 
+
+<!--
+  Copy the section link here.
+-->
+
+(Insert the permalink of the section heading in the Wagtail docs. Choose the most specific section / lowest-level heading to which your suggested change would apply. (In the docs, hover over the the heading to see the permalink icon.))
+
+### Suggestion type
+
+<!--
+  Select one of the following options by placing an 'x' inside the corresponding brackets, or just delete the other options. 
+-->
+
+|  x  | Type of change  | Fix/enhancement |
+| --- | ------------- | ------------- |
+| [ ] | Copy edit | Fix spelling or grammar errors, improve clarity
+| [ ] | Content correction | Correct inaccurate explanations, instructions, or code 
+| [ ] | New content | Add a new topic or provide more detailed instructions or examples 
+| [ ] | Reorganization | Move topic(s) 
+| [ ] | Other suggestion | Anything else
```

Comment:
```suggestion
```

I don’t think we need contributions to be precisely categorised like this, this should hopefully be clear enough from the title and the details.